docs/DeveloperGuide.adoc

```diff
@@ -723,15 +723,71 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
+|`* * *` |new user |import existing contacts |add contacts quickly when starting to use the App or reinstalling the App
+
+|`* * *` |frequent user |backup existing contacts as I add/remove them |restore a previous version of my contacts if I make a big mistake
+
 |`* * *` |user |add a new person |
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |sort persons by arrange the names in my contacts by Alphabetical order |locate a person easily
+
+|`* * *` |forgetful user |have prompts for right command syntax when I enter the wrong one |use the app more conveniently
```

Comment:
hmm... prompts for right command syntax doesn't help a forgetful user though,
do you mean suggest right commands for user? or autocomplete suggestion?